src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/fingerprints/SpoofSignatureVerificationFingerprint.kt

```diff
@@ -3,7 +3,7 @@ package app.revanced.patches.youtube.misc.fix.playback.fingerprints
 import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
 import org.jf.dexlib2.Opcode
 
-object ProtobufParameterBuilderFingerprint : MethodFingerprint(
+object SpoofSignatureVerificationFingerprint : MethodFingerprint(
```

Comment:
The method this fingerprint resolves to is setting a Protobuf parameter, is there any specific reason why it has been renamed?